java/client/src/main/java/glide/api/models/commands/WeightAggregateOptions.java

```diff
@@ -16,13 +16,20 @@ public class WeightAggregateOptions {
      * Option for the method of aggregating scores from multiple sets. This option defaults to SUM if
      * not specified.
      */
+    @RequiredArgsConstructor
     public enum Aggregate {
         /** Aggregates by summing the scores of each element across sets. */
-        SUM,
+        SUM("SUM"),
         /** Aggregates by selecting the minimum score for each element across sets. */
-        MIN,
+        MIN("MIN"),
         /** Aggregates by selecting the maximum score for each element across sets. */
-        MAX;
+        MAX("MAX");
+
+        private final String redisApi;
+
+        public String[] toArgs() {
+            return new String[] {AGGREGATE_REDIS_API, this.redisApi};
```

Comment:
I think you can avoid storing `redisApi` string
```suggestion
            return new String[] {AGGREGATE_REDIS_API, toString()};
```